src/frontend/package.json

```diff
@@ -13,7 +16,9 @@
     "gatsby-plugin-sharp": "2.4.5",
     "gatsby-source-filesystem": "2.1.48",
     "gatsby-transformer-sharp": "2.3.16",
+    "graphql-tag": "2.10.3",
     "highlight.js": "9.18.1",
+    "node-fetch": "2.6.0",
```

Comment:
Why do we need this here?

Reply:
required for node implementation of apolloclient. Otherwise it won't build properly with `npm run build`